packages/components/src/utils/breakpoints.ts

```diff
@@ -44,11 +44,20 @@ class Breakpoint {
         return breakpoint.minWidth <= innerWidth;
       });
 
+      if (!this.currentBreakpoint) {
+        return;
+      }
+
       if (!options.emitEvents) return;
 
-      Object.keys(this.currentBreakpoint)
-        .filter(key => this.currentBreakpoint[key] !== previousBreakpoint[key])
-        .forEach((key: BreakpointProperty) => this.dispatchEvent(key));
+      if (this.currentBreakpoint && previousBreakpoint) {
+        Object.keys(this.currentBreakpoint)
+          .filter(key => this.currentBreakpoint[key] !== previousBreakpoint[key])
+          .forEach((key: BreakpointProperty) => this.dispatchEvent(key));
+      } else if (this.currentBreakpoint && !previousBreakpoint) {
+        Object.keys(this.currentBreakpoint)
+          .forEach((key: BreakpointProperty) => this.dispatchEvent(key));
+      }
```

Comment:
You are already checking that `currentBreakpoint` is defined on line 47, so this could be simplified: 
```suggestion
      Object.keys(this.currentBreakpoint)
          .filter(key => !previousBreakpoint || this.currentBreakpoint[key] !== previousBreakpoint[key])
          .forEach((key: BreakpointProperty) => this.dispatchEvent(key));
```